<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiLap1214Class</class>
 <widget class="QMainWindow" name="QtGuiLap1214Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiLap1214</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="chooseBox"/>
        </item>
        <item>
         <widget class="QListWidget" name="eventsList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="titleLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Title</string>
              </property>
              <property name="buddy">
               <cstring>titleEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="titleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="descriptionLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
              <property name="buddy">
               <cstring>descriptionEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="descriptionEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="durationLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Duration</string>
              </property>
              <property name="buddy">
               <cstring>durationEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="durationEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="timeLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Time</string>
              </property>
              <property name="buddy">
               <cstring>timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="timeEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_10">
            <item row="0" column="0">
             <widget class="QLabel" name="PeopleLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>People going</string>
              </property>
              <property name="buddy">
               <cstring>peopleEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="peopleEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="linkLine">
              <property name="font">
               <font>
                <family>Verdana</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Link</string>
              </property>
              <property name="buddy">
               <cstring>linkEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="linkEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiLap1214.qrc"/>
 </resources>
 <connections/>
</ui>
